frontend/src/components/pages/admin/AssessmentEditorPage.tsx

```diff
@@ -74,6 +75,13 @@ const AssessmentEditorPage = (): React.ReactElement => {
     },
   });
 
+  const assessmentName = watch("name");
+  const isExisting = !!state;
+  usePageTitle(
```

Comment:
i feel like it actually might make more sense if we don't change page title when test name changes - and if the user deletes the entire test name, the title becomes `"Edit Assessment"` even though they are still in editing mode
<img width="1116" alt="image" src="https://github.com/uwblueprint/jump-math/assets/69697180/f4235414-7275-41eb-95c5-10e42f167bad">

Reply:
Wait @carissa-tang wym, just display "edit assessment" permanently? I felt like it might then be hard to disambiguate between different tests but maybe not?

Reply:
i mean like display the original test name permanently (if in editing mode)

Reply:
rather than having it change as the user changes the field value

Reply:
Oh, hmm okay. I guess that makes sense, updating on save